lodmill-rd/src/main/java/org/lobid/lodmill/PipeLobidOrganisationEnrichment.java

```diff
@@ -78,11 +78,11 @@
 	private enum VcardNs {
 		LOCALITY("http://www.w3.org/2006/vcard/ns#locality"), COUNTRY_NAME(
 				"http://www.w3.org/2006/vcard/ns#country-name"), STREET_ADDRESS(
-				"http://www.w3.org/2006/vcard/ns#street-address"), POSTAL_CODE(
-				"http://www.w3.org/2006/vcard/ns#postal-code"), EMAIL(
-				"http://www.w3.org/2006/vcard/ns#email"), VOICE(
-				"http://www.w3.org/2006/vcard/ns#Voice"), HOMEPAGE(
-				"http://www.w3.org/2006/vcard/ns#url");
+						"http://www.w3.org/2006/vcard/ns#street-address"), POSTAL_CODE(
+								"http://www.w3.org/2006/vcard/ns#postal-code"), EMAIL(
+										"http://www.w3.org/2006/vcard/ns#email"), VOICE(
+												"http://www.w3.org/2006/vcard/ns#Voice"), HOMEPAGE(
+														"http://www.w3.org/2006/vcard/ns#url");
```

Comment:
For enums, I usually format manually and use `@formatter:off` & `@formatter:on` comments, which are enabled in our custom formatter settings (just FYI).